R/splatPop-simulate.R

```diff
@@ -0,0 +1,849 @@
+#' splatPop simulation
+#'
+#' Simulate scRNA-seq count data using the splat model for a population of
+#' individuals with correlation structure.
+#'
+#' @param vcf vcfR object containing genotypes of samples to simulate.
```

Comment:
I was hesitant to do that because vcf objects can be very large. I get wanting to be able to recreate the whole simulation from the params object, but I've kept the gff, key, and vcf objects out of the param object for size concerns. If you don't think we should worry about its size, then I can include vcf or all three :)

Reply:
Maybe it's worthing checking how big each of these are (you can use `object.size()`)? I'm wondering how big they are compared to the expression matrices? That might help us decide if size is an issue or not.

Reply:
Haven't been able to come up with a good way to add info about the vcf to the params object automatically. My only thought is to have a vcf and gff container in the params object and give people the option to set it equal to a character, then in the vignette just encourage folks to put a helpful descriptor of the file or a URL to where it was downloaded from.